.github/workflows/ui_tests.yml

```diff
@@ -50,21 +50,10 @@ on:
         required: false
 
 jobs:
-  ui_tests:
-    name: 'Playwright Tests'
-    runs-on: [self-hosted, aws-ecs-16cpu-64mem-30m]
-    strategy:
-      fail-fast: false
-      matrix:
-        shard: [1, 2, 3, 4, 5]
-        shardTotal: [5]
-    timeout-minutes: 20
+  build_lab_and_extension:
+    name: 'Build Lab and Extension'
+    runs-on: ubuntu-latest
```

Comment:
I noticed the build step doesn't require `[self-hosted, aws-ecs-16cpu-64mem-30m]`